scripts/build/linux-cross

```diff
@@ -13,6 +13,7 @@ CROSS_OSARCH="linux/amd64 linux/arm linux/ppc64le"
 # linux/s390x
 
 echo "Building all linux binaries"
+# TODO(stevvooe): Remove gox. This just wraps environement variables.
```

Comment:
> You can already "build for x" by using script/build/binary with any environment variables you want.

That's pretty much the entire point.

Just a couple of days ago I was trying to test out some cross-build change (s390x support o something) and it was a massive pain. I don't think I even got it to work until well after the PR was merged.

Dependencies on parallel and gox was incredibly frustrating.